test/groups/test_special_euclidean_group.jl

```diff
@@ -388,7 +374,9 @@ using StaticArrays
         # If we apply another pullback, this seems to be right, so we have to check where we accidentally pushforward once too much?
         YL = inverse_retract(G, gL, kL, dirm)
         @test is_point(𝔤, YL; error = :error)
-        @test isapprox(𝔤, XL, YL)
+        # TODO: Check the above inverse retract, it seems either the pullback
+        # or the diff_left_compose (LLR case) has a sign error and gets the H component wrong
+        @test_broken isapprox(𝔤, XL, YL; atol = 1.0e-4, error = :error)
```

Comment:
This needs to be checked. Possible errors
* `pull_back_tangent` it seems a bit strange and could be documented with a proper formula
*  `diff_compose_left!` since it is used in the previous function. 

The symptom is that in the semidirect h-part instead of 1.0 we get a 1.46 - since that starts from 1.23 is might be a sign error on the h component as well.

Reply:
I have noch checked all internal functions, all their formulae are correct, So what might be is that the case here (LLR) got somehow confused by me with some other case regarding the differential (or the one of the right compose in the above inverse retracts push forward did) so that the “do not match”. But I would have to check probably all 8! cases again, which I do not feel able to.

Maybe someone with a fresh view might see where I confused 2 of the cases either in diff_left or diff_right compose.